### PR TITLE
Use "suggested query" when lid is omitted from CLI

### PR DESCRIPTION
**Pull request by clamdad**
SO far, this only updates the initial reporter configuration.  The reporter needs to be modified to add checks for new submissions, so it can update and stay in sync.

### CODE REVIEW

File: src/telliot_feed_examples/reporters/flashbot.py [hunk 1]
```diff
@@ -38,6 +38,7 @@ def __init__(
         chain_id: int,
         master: Contract,
         oracle: Contract,
+        sync_feed: bool,
```
Comment (clamdad) [inline src/telliot_feed_examples/reporters/flashbot.py:41]: This argument seems unused.

File: src/telliot_feed_examples/cli.py [hunk 2]
```diff
@@ -265,13 +282,12 @@ async def report(
...
         _ = input("Press [ENTER] to confirm settings.")
 
-        chosen_feed = LEGACY_DATAFEEDS[legacy_id]
-
         common_reporter_kwargs = {
             "endpoint": core.endpoint,
             "private_key": core.get_staker().private_key,
             "master": core.tellorx.master,
             "oracle": core.tellorx.oracle,
+            "sync_feed": sync_feed,
```
Comment (clamdad) [inline src/telliot_feed_examples/cli.py:290]: Debating the separate flag. Probably conflicts with `datafeed`, which is still a required argument to the reporters. 
 We should probably make `datafeed` an optional input.  If None, the default behavior would be to automatically choose the suggested feed.

Comment (oraclown) [reply src/telliot_feed_examples/cli.py:290]: Good idea

**Comment by oraclown** [comment]
@clamdad How's this?

**Comment by clamdad** [comment]
Looks great, ship it!

**Pull request merged**